src/hub.js

```diff
@@ -225,6 +230,7 @@ const onReady = async () => {
 
   const enterScene = async (mediaStream, enterInVR, hubId) => {
     const scene = document.querySelector("a-scene");
+    scene.style.cursor = "none";
```

Comment:
update hub.scss

Reply:
Currently we hide the cursor when the `render-target-loaded` event fires because `look-controls` adds the `a-grab-cursor` class to the `a-canvas` in the scene: https://github.com/aframevr/aframe/blob/master/src/components/look-controls.js#L344
Since we don't want to hide the cursor until we get past the initial menus, should I change this line to add a css class to the scene el and then having a selector for that class with a rule to disable the cursor in the scss file?
e.g. in hub.js `scene.classList.add(".no-cursor")` and in hub.scss `.no-cursor { cursor: none; }`